driver/src/util.rs

```diff
@@ -13,6 +14,10 @@ pub fn u128_to_u256(x: u128) -> U256 {
     U256::from_big_endian(&x.to_be_bytes())
 }
 
+pub fn u256_to_u128(x: U256) -> u128 {
```

Comment:
I am pretty sure this already exists [here](https://github.com/gnosis/dex-services/pull/249/commits/e07b1c8a25ec3db3144a04fa7d85f3e01aad60c9),. Did you cherry pick it? Test it?

Reply:
I cherry picked the code, forgot the tests.

Reply:
Now worries, it will be there eventually.